modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -164,7 +165,7 @@ public TableManager(
                         @Override public @NotNull CompletableFuture<?> onCreate(
                             @NotNull ConfigurationNotificationEvent<SchemaView> schemasCtx) {
                             try {
-                                ((SchemaRegistryImpl)tables.get(ctx.newValue().name()).schemaRegistry()).
+                                ((SchemaRegistryImpl)tables.get(ctx.newValue().name()).schemaView()).
```

Comment:
Could you explain removing `schemaRegistry` method from `AbstractTableView` and replacing it by `schemaView` in `TableImpl`, which returns `SchemaRegistry`despite of the name?

Reply:
Because the schemaView method is weightily used in our code before.
The one reason by that another method (schemaRegistry) was written is Alexander did not know about it already exists.